src/Avalonia.Controls/Platform/INativePopupImpl.cs

```diff
@@ -0,0 +1,12 @@
+using Avalonia.Metadata;
+
+namespace Avalonia.Controls.Platform;
+
+[Unstable]
+public interface INativePopupImpl
```

Comment:
Do we really need the extra interface? We could expose a new SetXxx method on `IPopupImpl`, which various implementations can ignore, like the existing `SetWindowManagerAddShadowHint`.